lib/instrumentations.js

```diff
@@ -19,7 +19,7 @@ module.exports = function instrumentations() {
     'express': { type: MODULE_TYPE.WEB_FRAMEWORK },
     'fastify': { type: MODULE_TYPE.WEB_FRAMEWORK },
     'generic-pool': { type: MODULE_TYPE.GENERIC },
-    '@grpc/grpc-js': { type: MODULE_TYPE.GENERIC },
+    '@grpc/grpc-js': { module: './instrumentation/grpc-js' },
```

Comment:
To be honest, I just cargo-culted this because I wanted it to load with `onResolve` like it does for Pino, because that's when Michael's original prototype loads, and I didn't want to deviate from his work.

Do you think that's ultimately unnecessary?